app/controllers/posts_controller.rb

```diff
@@ -12,4 +12,20 @@ def show
   def custom_post
     @post = Post.find(params[:id])
   end
+
+  def new
+    puts '^^^^^^^New is ^^'
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(user: @current_user, title: params[:title], text: params[:text])
```

Comment:
- [ ] [OPTIONAL] For creating any model attributes, always use strong parameters, this increases the security of data sent using forms.